distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso

```diff
@@ -153,8 +153,7 @@ all_same_column aggregates =
 
    If some columns cannot be resolved, `Nothing` is returned. Any issues are
    reported to the `Problem_Builder`.
-resolve_aggregate : Table -> Problem_Builder -> Aggregate_Column -> Aggregate_Column | Nothing
-resolve_aggregate table problem_builder aggregate_column =
+resolve_aggregate table:Table problem_builder:Problem_Builder aggregate_column:Aggregate_Column =
```

Comment:
```suggestion
resolve_aggregate table:Table problem_builder:Problem_Builder aggregate_column:Aggregate_Column -> Aggregate_Column | Nothing =
```

Reply:
As we are removing the other line we should add the return type as well.

Reply:
Yes `table` can be a `DB_Table` but we can't end up with a circle here.

We should create a conversion type `Table` and `DB_Table` can merge into but we haven't yet.

Reply:
I leave that for another libs PR then. I am fine with 5f9aa2510dd9f73c3009396d59b1a426acd0ed10 - everything is green.